docs/models_dev_guide.md

```diff
@@ -62,11 +62,12 @@ The following is an example contents of a `manifest.json` file:
     "models": [
         {
             "base_name": "VGG-16",
-            "base_filename": "vgg16_weights.npz",
+            "base_filename": "vgg16-imagenet.npz",
```

Comment:
Will the `flush` code handle the removal of the earlier model that now goes by a different name?  I suppose this may need to be done manually?

Reply:
It must be done manually. We shouldn't make a habit out of doing this, I just wanted to get us off the ground with a convention to include the dataset in the filename for future clarity.